### PR TITLE
Documentation nit fixes

### PR DESCRIPTION
**Pull request by jly36963**
I'm reading through the docs and source code, and I'm hoping to make more substantial contributions in the future.

### CODE REVIEW

File: contributing/style_guide.md [hunk 1]
```diff
@@ -185,9 +185,9 @@ export function pwrite(options: PWrite) {}
 
 ### Export all interfaces that are used as parameters to an exported member
 
-Whenever you are using interfaces that are included in the arguments of an
-exported member, you should export the interface that is used. Here is an
-example:
+Whenever you are using interfaces that are included in the parameters or return
```
Comment (jly36963) [inline contributing/style_guide.md:188]: I'm changing the overall meaning of this line. Let me know if my assumptions about its meaning are incorrect.

File: contributing/style_guide.md [hunk 2]
```diff
@@ -231,7 +231,7 @@ export function foo() {
 }
 ```
 
-It is important that documentation is easily human readable, but there is also a
+It is important that documentation is easily human-readable, but there is also a
```
Comment (jly36963) [inline contributing/style_guide.md:234]: Compound noun modifiers should be hyphenated.
Eg: `human-readable documentation`

File: contributing/style_guide.md [hunk 3]
```diff
@@ -354,15 +354,15 @@ export function foo(): string {
 
 `https://deno.land/std/` is intended to be baseline functionality that all Deno
 programs can rely on. We want to guarantee to users that this code does not
-include potentially unreviewed third party code.
+include potentially unreviewed third-party code.
 
 #### Document and maintain browser compatibility.
 
-If a module is browser compatible, include the following in the JSDoc at the top
+If a module is browser-compatible, include the following in the JSDoc at the top
```
Comment (kt3k) [inline contributing/style_guide.md:361]: I'd appreciate if you could do the same change in deno_std.

https://github.com/denoland/deno_std/search?q=browser+compatible

Comment (jly36963) [reply contributing/style_guide.md:361]: On it 😄

**Pull request merged**